docs/private-ddn/connector-deployment-resource-limits.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Connector Deployment Resource Limits
+sidebar_position: 5
+description:
+  "Understand how to configure resource limits for connectors in Hasura DDN to optimize performance and cost. Learn about CPU, memory, and deployment considerations."
+keywords:
+  - connector deployment
+  - resource limits
+  - cpu allocation
+  - memory allocation
+seoFrontMatterUpdated: true
+---
+
+# Connector Deployment Resource Limits
+
+## Introduction
+
+Connectors in Hasura DDN can request specific resources for their deployment. These include memory and CPU allocation. Understanding these limits ensures optimized performance and cost efficiency.
+
+## Default Resource Allocation
+
+By default, connectors have the following resource allocations:
+
+- **CPU**: 1 vCPU
+- **Memory**: 2GB RAM
+
+## Custom Resource Limits
+
+Resource limits can only be modified for **Private DDN**. For **Public DDN**, the resource limits are fixed to **1 vCPU and 2GB RAM**, and cannot be downsized or upscaled.
+
+### Public DDN
+
+- **Fixed CPU**: 1 vCPU
+- **Fixed RAM**: 2GB
+- CPU and memory cannot be customized for Hasura-provided connectors.
+- The limit and request are always set to the same value.
+
+### Private DDN
+
+- **Default CPU**: 1 vCPU
+- **Default RAM**: 2GB
+- The defaults, minimum, and maximum allowed values can be configured per data plane.
+
+## Memory and CPU Specifications
+
+Memory and CPU resources are specified using the same units as Kubernetes:
+
+- ### Memory Resource Units
+
+Limits and requests for memory are measured in bytes. Memory can be expressed as a plain integer or as a fixed-point number using one of the following quantity suffixes:
+
+- `E`, `P`, `T`, `G`, `M`, `k` (base-10 units)
+- `Ei`, `Pi`, `Ti`, `Gi`, `Mi`, `Ki` (power-of-two units)
+
+For example, the following values represent approximately the same amount of memory:
+
+- `128974848`, `129e6`, `129M`, `128974848000m`, `123Mi`
+
+> **Note:** Pay attention to the case of the suffixes. If you request `400m` of memory, this means `0.4` bytes, which is likely a mistake. Instead, you might intend to specify `400Mi` (400 mebibytes) or `400M` (400 megabytes).
+
```

Comment:
let's remove this bit, and just say that "we use the same unit specification for resources as kubernetes" and then link to the k8s docs there.

Reply:
any specific reason why we would want to do that? This section is inspired from kubernetes docs and links can always subject to change

Reply:
honestly, we don't even need the section because the resource units are a standard. But just in case, we can link to the k8s spec. we don't need to maintain a copy. 

Anyway, I'll leave it up to you. I've approved the PR.